gui.py

```diff
@@ -0,0 +1,145 @@
+import sys
```

Comment:
Please describe usage examples, including screenshots and feature behaviors. In mean the while, you have to update the readme.md and readme.zhcn.md as well for this change